README.md

```diff
@@ -1 +1,146 @@
\ No newline at end of file
+<a name="readme-top"></a>
+  
+<div align="center">
+ 
+  <img src="wave.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Author](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Portfolio: setup and mobile first <a name="about-project"></a>
+
+**Portfolio: setup and mobile first** is a Microverse portfolio project with version control system like git, HTML and CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Version Control System</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Frontend</summary>
+  <ul>
+    <li><a href="https://www.w3.org/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Beautiful mobile layouts**
+- **Site with nice styles**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- **A code editor of your choice (like vs code or Atom and so on)**
+ - **Version Control System (git is preferred)**
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Microverse-portfolio
+  git clone git@github.com:Fombi-Favour/Microverse-portfolio.git
+```
+
+### Usage
+
+Before you run the project, make sure the root file is **index.html**
+
+### Run tests
+
+To run tests, you can select the html file to be opened to any browser of your choice.
+
+
+### Deployment
+
+You can deploy this project on github following instructions here:
+[deploy website on github](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [Fombi-Favour](https://github.com/Fombi-Favour)
+- Twitter: [@FavourFombi](https://twitter.com/FavourFombi)
+- LinkedIn: [Fombi Favour](https://www.linkedin.com/in/fombi-favour/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [OPTIONAL] It would be nice to include the Future Features section in the README.md file. The **_Future Features_** section should contain 1-3 features that can be implemented in the future. 👍 😎